src/Catalyst.Cli/Options/ChangeDataFolderOptions.cs

```diff
@@ -0,0 +1,64 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System.Collections.Generic;
+using System.IO;
+using CommandLine;
+using CommandLine.Text;
+
+namespace Catalyst.Cli.Options
+{
+    /// <summary>
+    /// Class contains the options for the peer list command
+    /// </summary>
+    [Verb("changedatafolder", HelpText = "update node data folder")]
+    public sealed class ChangeDataFolderOptions : OptionsBase
+    {
+        private string _dataFolder;
+
+        /// <inheritdoc />
+        [Option('c', "datafolder", HelpText = "Data folder for the node.")]
+        public string DataFolder
+        {
+            get => _dataFolder;
+            set
+            {
+                if (Path.IsPathFullyQualified(value)) _dataFolder = value;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4594394)